multi-int-store/src/main/java/com/connexta/multiintstore/controllers/StorageController.java

```diff
@@ -36,16 +41,20 @@ public ResponseEntity store(
       @PathVariable String ingestId, @RequestBody(required = false) JsonNode body) {
     LOGGER.info("Received callback for ingestId {}", ingestId);
 
-    Object callback = validator.parse(body);
-
-    if (callback != null) {
+    try {
+      final Object callback = validator.parse(body);
```

Comment:
❓Could this method throw any exceptions? Where are these exceptions handled?

Reply:
validator.parse can throw an UnsupportedCallbackException and that should be it, the handleGeneralCallback throws all the runtime exceptions

Reply:
Might be worth moving the validator.parse out of this try/catch then